tests/python/openassetio/managerAPI/test_managerinterface.py

```diff
@@ -120,7 +149,28 @@ def displayName(self):
     def info(self):
         return {"stub": "info"}
 
+    def createState(self, _):
+        return StubManagerInterface.StubManagerState("a state")
+
+    def createChildState(self, parentState, _):
+        return StubManagerInterface.StubManagerState(f"a child state of {parentState.value}")
+
+
+class PersistenceStubManagerInterface(StubManagerInterface):
```

Comment:
[consider] This class is no longer used and can be removed.

Reply:
Similarly various methods of the `StubManagerInterface`

Reply:
Removed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/471/commits/184ab7db7ff1bed73c441967ac208aec40746422).